docs/madr/decisions/081-formalizing-security-update-policy-for-dependencies.md

```diff
@@ -0,0 +1,64 @@
+# MADR-081 Formalizing security update policy for dependencies
+
+* Status: accepted
+
+## Context and problem statement
+
+There is currently no formal policy for patching insecure dependencies in Kuma. Our goal has been to patch all vulnerabilities with CVSS 3.0 score ≥ 7.0 (High or Critical) across all supported versions. Supported versions are defined by the release branches listed in [`active-branches.json`](../../../active-branches.json).
+
+A consistent and scoped policy is needed to ensure we handle security updates effectively, especially across multiple release branches, without introducing unnecessary noise or instability.
+
+## Scope of coverage
+
+This policy applies to three categories of code:
+
+* Kuma's own source code
+* Third-party code directly linked by Kuma (e.g. Envoy, CoreDNS)
+* Third-party code bundled in the convenience Docker image (e.g. Python, Perl, cURL)
+
+For third-party code directly linked by Kuma, we only act on vulnerabilities that have a confirmed CVE.
```

Comment:
Sometimes, users report vulnerabilities sourced from GitHub security advisory database or other sources. Does that work?

Reply:
that means these are confirmed CVE